samples/js/js-eventsourced-shopping-cart/proto/shoppingcart_api.proto

```diff
@@ -56,22 +60,21 @@ service ShoppingCartService {
 
   rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      post: "/cart/{cart_id}/items/add"
-      body: "*"
+        post: "/cart/{user_id}/items/add"
+        body: "*"
     };
   }
 
   rpc RemoveItem(RemoveLineItem) returns (google.protobuf.Empty) {
-    option (google.api.http).post = "/cart/{cart_id}/items/{product_id}/remove";
+      option (google.api.http) = {
+          post: "/cart/{user_id}/items/{product_id}/remove";
```

Comment:
The semicolon needs to be removed here. Failed in the CI build when compiling the protobuf.

Reply:
```
Error: illegal name ';' (/home/circleci/project/samples/js/js-eventsourced-shopping-cart/proto/shoppingcart_api.proto, line 70)
```

Reply:
Thanks! I've removed it